apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/BrevDatafetcherVedtak.kt

```diff
@@ -38,6 +38,12 @@ internal class BrevDatafetcherVedtak(
             sak.sakType,
         )
 
+    suspend fun hentVilkaarsvurdering() =
+        brevdataFacade.hentVilkaarsvurdering(
+            behandlingId!!,
```

Comment:
Sikkert greit å ha en egen exception for dette

Reply:
Egentlig så kan kan sjekke dette tidligere i feks `omstillingsstoenadInnvilgelse`. da flere rest kallene trenger dette 👍🏻